flink-connector-mongodb-cdc/pom.xml

```diff
@@ -61,6 +61,14 @@ under the License.
                     <artifactId>connect-api</artifactId>
                     <groupId>org.apache.kafka</groupId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-databind</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-core</artifactId>
+                </exclusion>
```

Comment:
Please also exclude for module `flink-connector-debezium` ？

Reply:
`flink-connector-debezium` has already excluded its jackson dependency from Debezium and declares its version as `${jackson-bom.version}`. Is it necessary to exclude dependencies from `flink-connector-debezium` in its submodules?

Reply:
Thanks for your explanation, LGTM